_ui/_web_interface/views/system_control_card.py

```diff
@@ -65,7 +65,7 @@ def get_system_control_card_layout():
                         ],
                         className="field",
                     ),
-                    html.Div("Version 1.7.0"),
+                    html.Div(f"Version {SOFTWARE_VERSION} ({SOFTWARE_GIT_HASH})"),
```

Comment:
Using built metadata with version should be used via "+"
https://semver.org/#spec-item-10

Reply:
@dubrsl Fixed according to your suggestion, but with short hash instead of full one.